packages/reassure-cli/src/commands/measure.ts

```diff
@@ -1,18 +1,24 @@
-import { mkdirSync, rmSync } from 'fs';
+import { mkdirSync, rmSync, existsSync } from 'fs';
+import { resolve } from 'path';
 import { spawnSync } from 'child_process';
 import type { CommandModule } from 'yargs';
+import { run as runCompare } from './compare';
 
 const RESULTS_DIRECTORY = '.reassure';
 const RESULTS_FILE = '.reassure/current.perf';
 const BASELINE_FILE = '.reassure/baseline.perf';
 
 type MeasureOptions = {
   baseline: boolean;
+  compare: boolean;
 };
 
 export function run(options: MeasureOptions) {
+  console.log('\n❇️  Running performance tests:');
```

Comment:
TODO: make it a logger function so we can opt-out from logging some messages (make it an issue please)